src/util/vue-compat/index-3.js

```diff
@@ -4,5 +4,5 @@ const isVue2 = false;
 const isVue3 = true;
 const Vue2 = undefined;
 
-export * from 'vue';
+export { createApp, createSSRApp, h, version, nextTick } from 'vue';
```

Comment:
what's this for? did it accidentally bundle other parts of vue?

Reply:
Not sure why it happened but eslint couldn't find those, so I had to explicitly export them.